### DISCUSSION TITLE
Remove module name shadowing in sas.system.__init__

### DISCUSSION THREAD

**Discussion by pkienzle** [SasView/sasview#2340] [Ideas]
The name shadowing in `sas/system/__init__.py` such as `from .env import env` is confusing. With this style you cannot access the module without doing tricks like `import sys; env = sys.modules['sas.system.env']`.

**Comment by lucas-wilkins** [comment]
That's kind of the point of doing it.

**Comment by pkienzle** [comment]
From past experience this will cause difficulties for future maintainers.
